lib/gridfs-stream/upload.js

```diff
@@ -447,3 +486,15 @@ function writeRemnant(_this, callback) {
     checkDone(_this);
   });
 }
+
+/**
+ * @ignore
+ */
+
+function checkAborted(_this, callback) {
```

Comment:
Your link is the call to `checkDone()`, this function doesn't check for aborted itself, although it inserts in mongodb.
This call is after an async call, so the previous call to `checkAborted()` is not valid anymore.